src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -241,24 +240,18 @@ class Storage extends React.Component {
         editView: (value, { rowData }) => {
           const row = this.state.editing[rowData.id]
           const sizeGiB = row.size / (1024 ** 3)
-          const { invalidSizeValue } = row
 
           return <div className={style['disk-size-edit']}>
-            <FormGroup
-              validationState={invalidSizeValue && 'error'}
-              className={style['form-group-edit']}
-            >
+            <div>
               <FormControl
                 id={`${idPrefix}-${value}-size-edit`}
                 type='number'
-                min={this.props.minDiskSizeInGiB}
-                max={this.props.maxDiskSizeInGiB}
                 step={1}
                 value={sizeGiB}
                 className={style['disk-size-form-control-edit']}
                 onChange={e => this.handleCellChange(rowData, 'size', e.target.value)}
               />
-            </FormGroup>
+            </div>
             <span className={style['disk-size-edit-label']}>GiB</span>
             <div>
               <InfoTooltip id={`${idPrefix}-${value}-size-edit-info-tooltip`} tooltip={msg.diskEditorSizeCreateInfoTooltip()} />
```

Comment:
Could you please fix this message? Use "extend", not extends". Thanks.
Check also: https://github.com/oVirt/ovirt-web-ui/commit/bb683aaf83804f029c8c7fae7398bf082d726a83#diff-2ad0bd2e35e6b177ef9487ee23d8cbeed8061f518530fd769652907c66f4860cL208

Reply:
Sure :)